### DISCUSSION TITLE
Unable to highlight text in FabricJS textbox within a group

### DISCUSSION THREAD

**Discussion by clr7** [fabricjs/fabric.js#8858] [Q&A] [answered]
I'm trying to create sticky notes using FabricJS. The sticky note consists of a rectangular background and a textbox placed inside a group. I want to be able to highlight text in the textbox using the mouse (when textbox is in edit mode) but for some reason I cannot.

Live demo: https://jsfiddle.net/mp_10/8e5pxsq0/

**Comment by ShaMan123** [comment, marked as answer]
works on v6 ONLY
